apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -506,6 +506,10 @@ export const statistics = {
 		statistics.totalCustomRoles = await RolesRaw.findCustomRoles({ readPreference }).count();
 		statistics.totalWebRTCCalls = settings.get('WebRTC_Calls_Count');
 		statistics.uncaughtExceptionsCount = settings.get('Uncaught_Exceptions_Count');
+		statistics.pushEnabled = settings.get('Push_enable');
+
+		const defaultGateway = (await Settings.findOneById('Push_gateway'))?.packageValue;
```

Comment:
if you're looking for `packageValue` field only, add a projection:
```suggestion
		const defaultGateway = (await Settings.findOneById('Push_gateway', { projection: { packageValue: 1 } }))?.packageValue;
```